docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -440,6 +441,8 @@ Json Parameters:
           `Ulimits: { "Name": "nofile", "Soft": 1024, "Hard": 2048 }`
     -   **SecurityOpt**: A list of string values to customize labels for MLS
         systems, such as SELinux.
+    -   **StorageOpt**: Storage driver options per container. Options can be passed in the form
+	`[size:120G]`
```

Comment:
No, that would be if it was a struct (`{"key":"value"}`), this is an array (of strings)